helix-rest/src/test/java/org/apache/helix/rest/server/TestAuthValidator.java

```diff
@@ -54,9 +55,18 @@ public class TestAuthValidator extends AbstractTestClass {
   private String _mockBaseUri;
   private CloseableHttpClient _httpClient;
 
+  private static String CLASSNAME_TEST_DEFAULT_AUTH = "testDefaultAuthValidator";
+  private static String CLASSNAME_TEST_CST_AUTH = "testCustomAuthValidator";
+
+  @AfterClass
+  public void afterClass() {
```

Comment:
sorry for my ignorance, but shouldn't we be deleting Helix Cluster, rather than directly using zkclient to delete znodes?

Reply:
Thanks for the comments...I updated.